elastica/joint.py

```diff
@@ -222,51 +228,65 @@ def __init__(self, k, nu, kt):
             Damping coefficient of the joint.
         kt: float
             Rotational stiffness coefficient of the joint.
+        nut: float = 0.
+            Rotational damping coefficient of the joint.
+        use_static_rotation: bool = True
+            If True, the initial rotation offset between the two systems is recorded
+            and enforced throughout the entire simulation.
```

Comment:
```suggestion
            and enforced throughout the entire simulation. (default=True)
```

Reply:
Fixed in https://github.com/GazzolaLab/PyElastica/pull/135/commits/00ecf2f9f0b6ec6cbbe68af802052ed6e7edd932